src/main/java/mate/academy/bookstore/service/BookServiceImpl.java

```diff
@@ -36,4 +40,31 @@ public BookDto getBookById(Long id) {
                 new EntityNotFoundException("Can't find book by id = " + id));
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    @Override
+    public void updateById(Long id, CreateBookRequestDto requestDto) {
+        Book book = bookRepository.findById(id).orElseThrow(() ->
+                new EntityNotFoundException("Can't find book by id = " + id));
+        book.setTitle(requestDto.getTitle());
+        book.setAuthor(requestDto.getAuthor());
+        book.setIsbn(requestDto.getIsbn());
+        book.setPrice(requestDto.getPrice());
+        book.setDescription(requestDto.getDescription());
+        book.setCoverImage(requestDto.getCoverImage());
+        bookRepository.save(book);
```

Comment:
Why do you need to obtain the book from bd? 
Just map dto to book, set id and invoke save method